src/index.js

```diff
@@ -1,9 +1,22 @@
-import * as dataStrategies from './dataStrategies'
-import * as exportStrategies from './exportStrategies'
+export * from './utils'
+export * from './schemaToCSVTransforms'
+
 import * as utils from './utils'
+import results from './results'
+import terms_stats from './terms_stats'
+import * as schemaToCSVTransforms from './schemaToCSVTransforms'
+import * as csv from './fast-csv-wrapper'
+
+export {
```

Comment:
Are we just exporting these twice to give the user options?  Or is there something deeper I'm missing here?